packages/nimble-components/src/anchor-button/tests/anchor-button.stories.ts

```diff
@@ -0,0 +1,118 @@
+import type { Meta, StoryObj } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { html, when } from '@microsoft/fast-element';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import '../../all-components';
+import {
+    ButtonAppearance,
+    ButtonAppearanceVariant
+} from '../../patterns/button/types';
+import {
+    appearanceVariantDescription,
+    contentHiddenDescription,
+    endIconDescription,
+    iconDescription
+} from '../../patterns/button/doc-strings';
+import { hrefDescription } from '../../patterns/link/doc-strings';
+
+interface AnchorButtonArgs {
+    label: string;
+    href: string;
+    appearance: string;
+    appearanceVariant: string;
+    icon: boolean;
+    endIcon: boolean;
+    contentHidden: boolean;
+    disabled: boolean;
+}
+
+const metadata: Meta<AnchorButtonArgs> = {
+    title: 'Anchor Button',
+    decorators: [withXD],
+    parameters: {
+        docs: {
+            description: {
+                component:
+                    'An anchor button is a component with the visual appearance of a button, but it navigates like an anchor/link when pressed.'
```

Comment:
Could you add a note to the regular button docs that says something like: "If you want a button press to trigger navigation, use an `anchor-button` instead."?

Reply:
Done